### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by Mihndim2020**
For this pull request, I have styled the App using CSS3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # LeaderBoard-List-App
 The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
-![screenshot](images/leader2.png)
+![screenshot](images/leader3.png)
```
Comment (bellom) [inline README.md:7]: If you check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Live Demo Link`.

Kindly add Live Demo Link.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Mihndim2020  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

**1. Let's stick to the wireframe design provider for this project.**

image

Your design:
image

**Hint:** 
_- Kindly move the h2 tag with `Recent scores` and the `refresh` button to the top of the card._
_- Fix the `submit button, it is overlapping._

**2. Kindly check the comments under the review.**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Mihndim2020) [reply README.md:7]: Demo link added

**Comment by Mihndim2020** [comment]
All **requested changes** implemented. Thank you for your feedback.

File: src/style.css [hunk 2]
```diff
@@ -1,49 +1,51 @@
 html {
-  font-size: 12px;
+  font-size: 14px;
   box-sizing: border-box;
 }
 
 body {
-  display: flex;
-  margin: 0 auto;
   text-align: left;
```
Comment (MahmoudBakr23) [inline src/style.css:7]: - [x] Kindly fix the styling spacing issues with the app. See the image below for more information 😊🙏🏻
Screenshot_2021-07-30-00-01-33-31

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Mihndim2020 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mihndim2020** [comment]
Spacing issue fixed. Thanks for your feedback.

**Review by hemantso — APPROVED** [review]
Hi, @Mihndim2020, 

Good work on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**